things/postgres/channels_test.go

```diff
@@ -260,23 +262,24 @@ func TestMultiChannelRetrievalByThing(t *testing.T) {
 		},
 		"retrieve channels by non-existent thing": {
 			owner:  email,
-			thing:  "non-existent",
+			thing:  wrongID,
```

Comment:
Instead of breaking a valid test case, add UUID validation in the repository and return `ErrNotFound` if ID is in invalid format.